src/askui/tools/screen_switch_tool.py

```diff
@@ -0,0 +1,45 @@
+from askui.models.shared.tools import Tool
+from askui.tools.agent_os import AgentOs, DisplayInformation
+
+
+class ScreenSwitchTool(Tool):
```

Comment:
Can we

- call the file `display_tools.py` 
- add the tool for listing displays
- rename the tool here to `SetDisplayTool`

so that we are more consistent with our naming across the code base?